### ISSUE TITLE
Use maven artifacts

### ISSUE THREAD

**Issue by auscompgeek** [Lytigas/first-rust-competition#73]
There isn't a particular need to build a fork of allwpilib, so we should just use the upstream maven artifacts.

**Comment by Lytigas** [comment]
I agree. How often are those released? If they're only pushed up a few days before kickoff I might be more reluctant.

**Comment by auscompgeek** [comment]
Same cadence as allwpilib itself. In addition to the releases, development artifacts from master also get built.

**Comment by Lytigas** [comment]
I followed where `MavenArtifcats.md` pointed, but at the moment I can't seem to find the development packages; they don't seem to exist recently enough. I also can't find up-to-date NI libraries on there. Hopefully you can point out the right place to look.

**Comment by auscompgeek** [comment]
ni-libraries is at https://first.wpi.edu/FRC/roborio/maven/release/edu/wpi/first/ni-libraries/

From my understanding, for 2020 they've moved Maven artifacts to a different server because Java didn't agree with the TLS deployment on first.wpi.edu... my browser history seems to suggest it is at https://frcmaven.wpi.edu

**Comment by auscompgeek** [comment]
Filed https://github.com/wpilibsuite/allwpilib/issues/2068.

**Comment by Lytigas** [comment]
This would also be a good chance to update other things, like the compiler and HAL to 2020. But, I'm not in the beta. How do you handle this for pyfrc?

**Comment by auscompgeek** [comment]
Short answer is we don't very well. At the very least we could get simulation working for everyone, but testing on a real robot would indeed require the beta roboRIO image.

**Comment by Lytigas** [comment]
I do know some people in the beta, I might be able to get access. Anyway, 2019 development stopped with `2019.1.4`, so it seems like we should stop 2019 development with `0.4.0` and at least get things compiling and react to breaking changes made in the HAL.
